src/Hedgehog/Extras/Test/Golden.hs

```diff
@@ -43,14 +44,21 @@ mGoldenFileLogFile :: Maybe FilePath
 mGoldenFileLogFile = IO.unsafePerformIO $
   IO.lookupEnv "GOLDEN_FILE_LOG_FILE"
 
--- | Whether the test should create the golden files if the file does ont exist.
+-- | Whether the test should create the golden files if the file does not exist.
 createFiles :: Bool
 createFiles = IO.unsafePerformIO $ do
   value <- IO.lookupEnv "CREATE_GOLDEN_FILES"
   return $ value == Just "1"
 
--- | Diff contents against the golden file.  If CREATE_GOLDEN_FILES environment is
--- set to "1", then should the gold file not exist it would be created.  If
+-- | Whether the test should recreate the golden files even if the files do exist.
+recreateFiles :: Bool
```

Comment:
I forgot we had `createFiles`. We can just delete the golden files and set `CREATE_GOLDEN_FILES=1`.